### PR TITLE
 Rename project tab to Teams

### LINKED ISSUE #89: Rename 'Projects' tab to 'Teams' in Profile Page
Currently, the 'Projects' tab on the profile page shows the teams the user is a part of, regardless of whether the team actually submitted a project or not. So, the name 'Projects' is a bit confusing since it doesn't serve the purpose.

### Task

- Rename 'Projects' tab to 'Teams' in Profile Page

### PR DESCRIPTION
**Pull request by PuneetDabral**
**Fixes**

Fixes #89  by @shubhanshu02 

**Description**

**Checklist:**

-   [ ] I have read the contribution guidelines in the repository.
-   [ ] My branch is up-to-date with upstream branch.
-   [ ] My pull request has a descriptive title (not a vague title like "Update package.json").
-   [ ] My code follows the established code style of the repository.
-   [ ] I have tried running the project locally and verified that there are no visible errors.

### CODE REVIEW

**Comment by PuneetDabral** [comment]
thanku sir  for giving me this opportunity please review these changes and merge this ...

File: pages/profile/[username].js [hunk 1]
```diff
@@ -203,7 +202,7 @@ function Profile() {
                                         </Nav.Item>
                                         <Nav.Item>
                                             <Nav.Link eventKey="projects">
-                                                Projects
+                                                Teams
```
Comment (shubhanshu02) [inline pages/profile/[username].js:205]: Other than simply changing the tab name, the `eventKey` attribute must be changed. And similarly to the Tabs where this event key is being accessed.

**Review by shubhanshu02 — CHANGES_REQUESTED** [review]

**Comment by shubhanshu02** [comment]
Also, do fill the pull request description with the appropriate issue number and other details.

**Comment by PuneetDabral** [comment]
now its ok ...sorry sir for extra commit actually i am in beginner phase ...so i need all of your support so i can learn more & more

**Review by shubhanshu02 — CHANGES_REQUESTED** [review]
Looks good to me :+1:. 

To resolve the merge conflict, can you rebase your branch with the main branch of this repository? The steps would be:

```
git remote add upstream https://github.com/COPS-IITBHU/hackalog-frontend.git
git pull upstream main --rebase
git push origin Puneet -f
```

**Comment by PuneetDabral** [comment]
wait  sir i will ...!!

**Comment by PuneetDabral** [comment]
sir it will show error .....unable to access 'https://github.com/PuneetDabral/hackalog-frontend.git/': Could not resolve host: github.com

**Comment by shubhanshu02** [comment]
If I'm not wrong the last command must be giving this error, right?

**Comment by PuneetDabral** [comment]
now check it sir ...

**Review by shubhanshu02 — APPROVED** [review]
Ya, now it's fine. Thanks.

Btw, don't call me 'sir'. I'm still a student. :wink:

**Comment by PuneetDabral** [comment]
thank you very much i learn alot from you ..can you give me more (good first issue) so i can learn more ....thanku once again

**Comment by shubhanshu02** [comment]
I'll see and let you know tomorrow. Thanks

**Comment by PuneetDabral** [comment]
ok sir i am waiting you can give me. more good first issue of beginner level from this or may be other repo....

**Comment by PuneetDabral** [comment]
sir any good first issue?..so i can learn more from you...

**Comment by shubhanshu02** [comment]
Sorry, was really busy in the last two days. How about #69?

**Comment by PuneetDabral** [comment]
its ok sir.....basically sir. i am beginner so learn react. if you help me or teach me . i will definitely solve this issue and some other also  ...

**Comment by shubhanshu02** [comment]
You can refer to the documentation of the Effect hook (https://reactjs.org/docs/hooks-effect.html)

**Pull request merged**